index.html

```diff
@@ -0,0 +1,104 @@
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title> Lynn's portofolio</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css">
+</head>
+<body>
+    <header>
+        <h2>John Doe</h2>
+
+        <nav>
+          <ul class="menu">
+            <li><a href="#">Home</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contacts</a></li>
+          </ul>
+        </nav>
+      </header>
+      
+    <main class="main">
+        <h1>I'm Laura Jong </h1>
+         <p>Software Developer</p>
+        <p>I can help you  to build a products , feature or website Look through some of my work and experience!<br/> If you like what you see and have a project you need coded,  don’t hesitate to contact me.</p>
+
+   </main>
+       
+   
+        <footer class="footer section bd-container">
+          <div class="footer__container bd-grid">
+              <div class="footer__content">
+
+              <div class="footer__content">
+                  <h3 class="footer__title">Language</h3>
+                  <ul>
+                    <li>HTML</li>
+                    <li>CSS</li>
+                    <li>JAVASCRIPT</li>
+                    <li>RUBY</li>
+                    <li>PHP</li>
+                    <li>DART</li>
+                </ul>
+              </div>
+
+              <div class="footer__content">
+                  <h3 class="footer__title">Frameworks</h3>
+                  <ul>
+                    <li>Boostrap</li>
+                    <li>Ruby or Rails</li>
+                    <li>Rspec</li>
+                    <li>Capybara</li>
+                    <li>Selenium</li>
+                </ul>
+              </div>
+
+              <div class="footer__content">
+                  <h3 class="footer__title">Skills</h3>
+                  <ul>
+                      <li>Github</li>
+                      <li>Gitlab</li>
+                      <li>Codepen</li>
+                      <li>Codekit</li>
+                      <li>Terminal</li>
+                  </ul>
+              </div>
+          </div>
+          </div>
+        </footer>
+      
+
+<center>
+  <p><span>&darr;</span>work</p>
+</center>
+
+<section class="link-list">
+  <ul>
+    <li>
+      <a href="https://wwww.com/" class="link-with-icon">
+        <i class="fab fa-github"></i>
+      </a>
+    </li>
+    <li>
+      <a href="https://www.com/in/" class="link-with-icon">
+        <i class="fab fa-linkedin"></i>
+      </a>
+    </li>
+    <li>
+      <a href="https://www.com/" class="link-with-icon">
+        <i class="fab fa-instagram"></i>
+      </a>
+    </li>
+    <li>
+      <a href="https://www.com" class="link-with-icon">
+        <i class="fab fa-facebook"></i>
+      </a>
+    </li>
+  </ul>
+</section>
```

Comment:
<ul> tag is already a block, no need to wrap it into a section tag. Instead of section class="link-list" ul li 
we can have ul class="link-list" li